test/cql-pytest/conftest.py

```diff
@@ -29,6 +29,7 @@
 from cassandra.policies import RoundRobinPolicy
 import ssl
 
+import test_config
```

Comment:
Again, I like the idea that all the non-standard pytest-modifying trickery happens just in one place (conftest.py). This makes it easier to read and understand how our pytest environment differs from the standard.  I don't see a reason to split it into three files (conftest.py, test_config and custom_decorators.py). If you feel otherwise, please try to convince me.